index.html

```diff
@@ -0,0 +1,162 @@
+					</a>
+				</h2>
+				<div class="center">
+					<a href="#">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" width="980" height="64" alt="Watch the “Your Verse” film and explore the stories">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_46pt.png" width="980" height="112" alt="Watch the “Your Verse” film and explore the stories">
+					</a>
+				</div>
+			</header>
+    </section>
+  </main>
+  
+  <div class="promo">
+    <ul>
+      <li>          
+        <a class="promo1" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" width="364" height="200" alt="iPhone 5s. Forward thinking.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo2" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" width="364" height="200" alt="iPhone 5c. For the colorful.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo3" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" width="364" height="200" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo4" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" width="364" height="200" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+        </a>    
+      </li>
+    </ul>
+  </div>
+
+  <footer>
+    <div class="wrapper">
+      <p class="left">Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>, or find a <a href="#">reseller</a>.</p>
+      <ul class="piped right">
+        <li class="first">
+          <a href="#">Site Map</a>
+        </li>
+        <li>
+          <a href="#">Hot News</a>
+        </li>
+        <li>
+          <a href="#">RSS Feeds</a>
+        </li>
+        <li>
+          <a href="#">Media Info</a>
+        </li>
+        <li>
+          <a href="#">Environment</a>
+        </li>
+        <li>
+          <a href="#">Job Opportunities</a>
+        </li>
+        <li>
+          <a href="#">Contact Us</a>
+        </li>
+        <li class="worldwide">
+          <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/global/elements/flags/16x16/usa.png" alt="Choose your country or region" height="16" width="16"></a>
```

Comment:
**height="16" width="16"**
Add the right selectors for width and height in your CSS file